src/infra/nixl_descriptors.cpp

```diff
@@ -409,10 +409,9 @@ nixl_status_t nixlDescList<T>::serialize(nixlSerDes* serializer) const {
 
 template <class T>
 void nixlDescList<T>::print() const {
-    std::cout << "LOG: DescList of mem type " << type
-              << (sorted ? "sorted" : "unsorted") << "\n";
+    NIXL_INFO << "DescList of mem type " << type
```

Comment:
looking at the code it should be printed with default log level. I do not think error or warning is appropriate, so maybe just use cout?

Reply:
maybe keep nixlDescList as is for now (use cout), going forward print() should be deprecated in favor of operator<< IMO